src/main/java/uk/gov/di/resources/WellKnownResource.java

```diff
@@ -1,31 +1,47 @@
 package uk.gov.di.resources;
 
+import com.nimbusds.jose.jwk.JWKSet;
+import uk.gov.di.configuration.OidcProviderConfiguration;
+import uk.gov.di.services.TokenService;
+
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import static java.text.MessageFormat.format;
 
 @Path("/.well-known/")
 public class WellKnownResource {
+
+    private TokenService tokenService;
+    private OidcProviderConfiguration configuration;
+
+    public WellKnownResource(TokenService tokenService, OidcProviderConfiguration configuration) {
+        this.tokenService = tokenService;
+        this.configuration = configuration;
+    }
+
     @GET
     @Path("/openid-configuration")
     @Produces("application/json")
     public String openIdConfiguration() {
         
         return """
         {
-            "issuer": "https://di-auth-oidc-provider.london.cloudapps.digital",
-            "authorization_endpoint": "https://di-auth-oidc-provider.london.cloudapps.digital/authorize",
-            "token_endpoint": "https://di-auth-oidc-provider.london.cloudapps.digital/token",
+            "issuer": "<baseUrl>",
```

Comment:
Another thing that got me was that we set the issuer in the oidc-provider.yml to `di-auth-oidc-provider'. This value is used for setting the issuer in the ID token. As these values were different, the python lib was unable to validate the issuer of the ID token